maven/api-maven-archive/src/main/java/org/jboss/shrinkwrap/resolver/api/maven/archive/assembler/ArchiveMavenAssembler.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * JBoss, Home of Professional Open Source
+ * Copyright 2012, Red Hat Middleware LLC, and individual contributors
```

Comment:
Copy-pasted from MavenImporter class. I didn't know about it - is there some document/email/article about it?